blocks/commerce-wishlist/commerce-wishlist.js

```diff
@@ -60,10 +66,29 @@ export default async function decorate(block) {
     'start-shopping-url': startShoppingURL = '',
   } = readBlockConfig(block);
 
+  const getProductLink = (product) => (product ? rootLink(`/products/${product.urlKey}/${product.sku}`) : rootLink('#'));
+
   await wishlistRenderer.render(Wishlist, {
     routeEmptyWishlistCTA: startShoppingURL ? () => rootLink(startShoppingURL) : undefined,
     moveProdToCart: cartApi.addProductsToCart,
     routeProdDetailPage: (product) => rootLink(`/products/${product.urlKey}/${product.sku}`),
```

Comment:
should we use the new `getProductLink()` function you created? It will consolidate links to a single source.